### ISSUE TITLE
Use full version for hi-res tablets

### ISSUE THREAD

**Issue by inliquid** [esotalk/esoTalk#323]
Some tablets has resolution higher then some desktops. Using mobile version on them is something very strange. I think it make sense for esoTalk to enable mobile skin depending on screen resolution, and not operating system (like Android or iOS). Or differentiate (hi-res) tablets from smartphones.

**Comment by jgknight** [comment]
Using screen resolution may be too arbitrary.  I would propose adding a user setting, or adding an option within the session, on whether or not to use mobile/desktop versions of the site.

**Comment by tvb** [comment]
Agreed. However some degree of different screen resolutions other then the mobile we have now is probably wise to implement.
